### DISCUSSION TITLE
Quiver Upgrade

### DISCUSSION THREAD

**Discussion by Cynan** [P3pp3rF1y/SophisticatedBackpacks#169] [Ideas]
Suggestion - Quiver Upgrade that adds a few extra slots to the right that bows can pull arrows from.
First level could be a single slot and the Advanced Quiver Upgrade could add one or more extra slots for additional arrow storage.

**Comment by TheDeviantCrafter** [comment]
The refill upgrade can keep a stack of arrows in the inventory topped up. Is that sufficient?

**Reply by Cynan** [reply]
I like the refill ability but but it was actually triggers by accidentally dumping arrows out of inventory when returning with loot.  A quiver that was separate would prevent that.  Not a huge issue, just a little quality of life improvement.
